cli-toolchain/quickstart.md

```diff
@@ -450,6 +450,22 @@ What happens behind the scenes?
   You can get the `AWSCLOUDFORMATIONCONFIG` from the `team-provider-info.json` file from your existing Amplify project in the `amplify/` directory of the project.
 
 
+## Mocking and Testing
+
+It is highly recommended that you have completed the Getting Started section of Amplify setup before using local mocking.
+
+- [JavaScript Getting Started](../js/start.md)
+- [Android Getting Started](../android/start.md)
+- [iOS Getting Started](../ios/start.md)
+
+Amplify supports running a local server for mocking and testing your application before pushing to the cloud with certain categories, including API (AWS AppSync), Storage (Amazon DynamoDB and Amazon S3), Functions (AWS Lambda), and Hosting. After running `amplify init` you can run the following to start a mock server:
```

Comment:
Just to make sure on the details .... For mocking in Storage we only support S3 and not DynamoDb, right ?

Reply:
Technically we support both since AppSync uses the local DynamoDB mock. There is a note in the Storage section which is explicit about this to avoid customer confusion.